src/Akka.Streams.Kafka/Settings/ProducerSettings.cs

```diff
@@ -33,13 +34,23 @@ public sealed class ProducerSettings<TKey, TValue>
         public TimeSpan EosCommitInterval { get; }
         public IImmutableDictionary<string, string> Properties { get; }
 
+        /// <summary>
+        /// Gets property value by key
+        /// </summary>
+        public object this[string propertyKey] => this.Properties.GetValueOrDefault(propertyKey);
+        
+        public string GetProperty(string key) => Properties.GetValueOrDefault(key, null);
+        
         public ProducerSettings<TKey, TValue> WithBootstrapServers(string bootstrapServers) =>
             WithProperty("bootstrap.servers", bootstrapServers);
 
         public ProducerSettings<TKey, TValue> WithProperty(string key, string value) =>
             Copy(properties: Properties.SetItem(key, value));
 
-        public ProducerSettings<TKey, TValue> WithProperties(IDictionary<string, string> properties)
+        public ProducerSettings<TKey, TValue> WithProducerConfig(ProducerConfig config)
```

Comment:
Convenience method to pass `Confluent.Kafka.ProducerConfig` directly into our `ProducerSettings`